platform/openide.loaders/src/org/openide/awt/Toolbar.java

```diff
@@ -617,4 +623,23 @@ public Icon getIcon() {
         }
     }
 
+    /**
+     * A button that provides a default icon when no text and no custom icon have been set.
+     */
+    private static class DefaultIconToggleButton extends JToggleButton {
```

Comment:
I think you should bump the specification version of the `openide.awt` module and make sure your modules depend on the new version.

A note in `apichanges.xml` might be appropriate as well.

Reply:
Version upgraded and a note added to apichanges. It's not a change in API interfaces, just extension in the behavior.